charts/pulsar/templates/broker/broker-statefulset.yaml

```diff
@@ -58,6 +58,10 @@ spec:
       imagePullSecrets:
       - name: {{ .Values.imagePullSecrets }}
       {{- end }}
+      {{- if .Values.broker.imagePullSecrets }}
+      imagePullSecrets:
+      - name: {{ .Values.broker.imagePullSecrets }}
```

Comment:
With this format, only one secret can be set here

Reply:
Never mind. It's ok here for the broker

Reply:
yeah, either set different imagePullSecret for each component, or use a central imagePullSecret for all of them